backend/app/services/session.py

```diff
@@ -0,0 +1,8 @@
+from sqlmodel import Session
```

Comment:
what is this file for?

Reply:
fixed

Reply:
what did you fix?  this file should not be in the repo

Reply:
deleted